src/lib/replication/service/repl_service_impl.cpp

```diff
@@ -52,7 +52,6 @@ ReplicationServiceImpl::create_repl_dev(uuid_t group_id, std::set< std::string,
     superblk< repl_dev_superblk > rd_sb{"repl_dev"};
     rd_sb.create(sizeof(repl_dev_superblk));
     rd_sb->gid = group_id;
-
```

Comment:
Nit: I would like to avoid contiguous pieces of lines of code and would rather have this new line. It gives a clear block of code.

Reply:
updated